doc/Reference.tex

```diff
@@ -951,3 +951,15 @@ \section{Dimensional Analysis}
 All function objects require dimensionless inputs. You can use dimensional 
 analysis to prevent incorrectly feeding a degree measurement into a sin, by 
 requiring them to be multiplied by a radiansPerDegree parameter.
+
+\section{Bookmarks}
+
+Bookmarks are a useful feature for saving the system at a particular point of the
+simulation. To create a new bookmark, click on the ``Bookmark'' tab in the top
+left-hand corner (in-between ``Edit'' and ``Insert'') and select ``Bookmark this
+position''. The program will provide a dialogue box to enter in a name for the new
+bookmark. After creating the bookmark, all user-created bookmarks can be seen in the
+Bookmarks menu. To delete a bookmark, simply select ``Delete'' from the Bookmarks
+menu and select the desired bookmark. To open an existing bookmark, select one 
+from the menu, and press play to continue running the simulation from that point 
```

Comment:
A little misleading - bookmarks refer to position and zoom level of the canvas, not where the simulation is at.